.github/pull_request_template/hotfix.md

```diff
@@ -0,0 +1,27 @@
+# 🚨 Hotfix
+
+## Checklist
+- [ ] Critical issue: #___
+- [ ] Maintainer approval obtained
+- [ ] Branch: `hotfix-[issue]-[desc]`
+- [ ] Title: `<component>: <message> Fix #<issue>`
+- [ ] Minimal, targeted change
+- [ ] Thoroughly tested
+- [ ] Ready for immediate release
+- [ ] CI passing
+
+## Description
+
+*Note: Skip this section if the GitHub issue already describes the critical problem, impact and urgency clearly.*
+
+**Problem & Impact:**
+*What's critically broken and who/what is affected*
+
+**Fix:**
+*Minimal solution applied*
+
+**Risk & Rollback:**
+*Why this is safe to deploy immediately and how to revert if needed*
```

Comment:
I feel like reverting would always be 'revert the commits' in the context of the repo and 'revert to previous Rucio version' in the context of the deployment